test/index.spec.js

```diff
@@ -84,4 +84,15 @@ describe('immutableStateInvariantMiddleware', () => {
       dispatch({type: 'SOME_ACTION', x});
     }).toNotThrow();
   });
+
+  it('works correctly with NaN', () => {
```

Comment:
Can we move these tests into the proper file? Since this is a change made in `trackForMutations.js`, the place to write tests for it is `/test/trackForMutations.spec.js`. Also, take a look there that we have the tests cases in an object that describes them, both for mutations and for non mutations. You should add an entry to the nonMutations object here: https://github.com/matthieu-foucault/redux-immutable-state-invariant/blob/b21595635a88d3c4e8223f2bb7203c140415fb72/test/trackForMutations.spec.js#L175